lib/easy_stalk/client.rb

```diff
@@ -35,18 +35,18 @@ def self.enqueue(
 
     def self.create_worker_pool(tubes, config=EasyStalk.configuration)
       raise "beanstalkd_urls not specified in config" unless config.beanstalkd_urls
-      conns = config.beanstalkd_urls.shuffle
-      i = 0
       # workers should only ever run a single thread to talk to beanstalk;
       # set the pool size to "1" and the timeout low to ensure that we don't
       # ever violate that
       instance = EzPool.new(size: 1, timeout: 1, max_age: config.worker_reconnect_seconds, disconnect_with: lambda{ |client| client.close }) do
         # rotate through the connections fairly
-        client = Beaneater.new(conns[i])
-        i = (i + 1) % conns.length
-        client.tubes.watch!(*tubes)
+        beanstalkd_url = config.beanstalkd_urls.sample
```

Comment:
I don't think this change makes sense; it seems better to guarantee fairness the way it used to.

Reply:
We have a set of jobs that we want to exclude from specific worker types and additional ones may be added in the future. It isn't a ton of tubes, but we want to ensure if someone adds a new one it will automatically be excluded from specific workers.

Reply:
what do you mean by better? if you have multiple worker processes following the exact same pattern reconnecting to beanstalk instances then you are more likely to have starved servers. this rotation change is somewhat of an imporvement making sure that each worker process is choosing a randomish server to connect to. 

as with the glob change, this client library is just giving the option to its users. if the user sees fit to have multiple beanstalk tubes following a pattern and want to initiate them specifically then they should have the option to say `easy_stalk:work["myservice.pattern-*"]

Reply:
The previous method guaranteed that every server would receive an equal share of time.

This method only probabilistically does so.

Reply:
fair point, I added back the original rotation but added a random `i` initiation.

Reply:
didn't the now-removed `config.beanstalkd_urls.shuffle` provide a superior level of randomness, since its not just origin but also ordering that gets shuffled?